check-for-missing-checkstyle-rule.py

```diff
@@ -30,22 +30,32 @@ def check_output_discarding_stderr(*args, **kwargs):
         return subprocess.check_output(*args, stderr=devnull, **kwargs)
 
 
+def try_decode(s):
+    if type(s) == bytes:
+        return s.decode()
+    return s
+
+
+def lmap(func, seq):
```

Comment:
Is there a reason why we need to abstract `list(map(func, seq))` into `lmap`?

Reply:
@lolski in this case, it's just shorter to read